domains/chiristo.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "nimalchrist",
+        "email": "chiristo.dev@gmail.com"
+    },
+    "record": {
```

this should be "records" (with "s").
```suggestion
    "records": {
```